content/tracing/product_specs/distributed_tracing.md

```diff
@@ -0,0 +1,162 @@
+---
```

Comment:
To really explain how this work, we might want to explain more about what is going on underneath? (more especially that we propagate trace_id/span_id to make it work).

Examples like http://pypi.datadoghq.com/trace/docs/#id2 make it more explicit.

Reply:
The idea here is just to give a global overview of what distributed tracing is, what's the end-goal of it, what's the DD UI attached to it.

But you are right, we can give more information about how we propagate trace_id and span_id to make it work in order to make people realize what "manual" instrumentation is..